src/components/RedirectionNotificationModal.sc.js

```diff
@@ -0,0 +1,96 @@
+import Box from "@mui/material/Box";
+import styled from "styled-components";
+
+const MyBox = styled(Box)`
```

Comment:
there is a lot of duplication between this and the other MyBox defined for the other modal. 
do you think we could factor them out into a single MyBox? 
And maybe we should think about giving them a more meaningful name than MyBox :)